tests/Fixer/PhpUnit/PhpUnitTestClassRequiresCoversFixerTest.php

```diff
@@ -448,4 +448,62 @@ public static function provideFix82Cases(): iterable
                 PHP,
         ];
     }
+
+    /**
+     * @dataProvider provideFixUnit11Cases
+     *
+     * @requires PHP 8.3
+     */
+    public function testFixUnit11(string $expected, ?string $input = null): void
+    {
+        $this->doTest($expected, $input);
+    }
+
+    public static function provideFixUnit11Cases(): iterable
+    {
+        yield 'already with attribute CoversMethod' => [
+            <<<'PHP'
+                <?php
+                #[PHPUnit\Framework\Attributes\CoversMethod(Foo::class)]
```

Comment:
In terms of testing the fixer it does not matter, but this is not a valid usage of `CoversMethod` 🙂.

```suggestion
                #[PHPUnit\Framework\Attributes\CoversMethod(Foo::class, 'bar')]
```

Same in other cases.

Reply:
Ah yes, sorry